environments/adapters/frontend.py

```
@@ -16,15 +17,57 @@
 )
 
 
+def _dogfood_asset_check(name: str, path: Path) -> Check:
+    if not path.is_file():
+        return Check(
+            name,
+            "preparation_required",
+            f"dogfood Frontend asset is missing: {path}",
+            False,
```

Ruff (0.16.1)

[warning] Boolean positional value in function call

(FBT003)